### PR TITLE
WIP: Multiple imports with jQuery dependencies.

### PR DESCRIPTION
**Pull request by raghunayyar**
- [x] remove jquery fileupload.
- [x] fix error on multiple upload.
- [x] remove the calendar from the list if import is successful, spinner for the time being?
- [x] remove the calendar from the list if hit remove button

### CODE REVIEW

**Comment by raghunayyar** [comment]
@georgehrke as discussed, WIP.

**Comment by georgehrke** [comment]
Removing lots of stupid dependencies :clap::clap::clap:

cool!

File: js/app/controllers/settingscontroller.js [hunk 1]
```diff
@@ -29,40 +29,36 @@
...
+		};
 
 		$scope.importcalendar = function (id) {
 			$scope.calendarid = id;
 		};
 
 		$scope.pushcalendar = function (id, index) {
+				var importinput = document.getElementById('import');
+				var reader = new FileReader();
+      	$scope.filescontent = importinput.files[0];
```
Comment (georgehrke) [inline js/app/controllers/settingscontroller.js]: please fix the whitespace dude

**Comment by georgehrke** [comment]
please rebase

**Comment by georgehrke** [comment]
please fix the issues I mentioned and rebase. I'll take a look afterwords.

**Comment by raghunayyar** [comment]
Done, @georgehrke, now your turn ;)

**Comment by raghunayyar** [comment]
@georgehrke I think we should support it till the DAV app is up and running.

**Comment by raghunayyar** [comment]
And since its already there with two really small fixes, why not fix them and ship it?

**Comment by georgehrke** [comment]
The dav app is update and running :)
We just need to make our app an caldav client.

**Comment by georgehrke** [comment]
But okay, I'll check. This is rather important for testing purposes.

**Pull request closed without merging**